### ISSUE TITLE
prevent mstar to sample <= 0

### ISSUE THREAD

**Issue by sharonxuesong** [California-Planet-Search/radvel#115]
Line 332 in driver.py:
    mstar = np.random.normal(
        loc=P.stellar['mstar'], scale=P.stellar['mstar_err'], 
        size=len(chains)
        )

When mstar_err is too big, mstar may go to zero or below. This will result in null columns in the multiplied chains for derived parameters, and radvel would fail to plot the corner plot for derived parameters.

cheers,
sharon

**Comment by petigura** [comment]
I'm in favor of throwing a warning, but not an error when this happens. If a few chains samples of mstar are negative it's not a big deal, but I'm concerned that if we just clip out all negative chains, we'll produce unintuitive behavior down the road.

Erik

**Comment by bjfultn** [comment]
Fixed in commit 8429768d530

I now replace negative stellar masses with their absolute values and throw the following warning.
```
WARNING: 12565 (0.16 %) of Msini samples are NaN. The stellar mass posterior may contain negative values. Interpret posterior with caution.
```
